cmd/benchcheck/benchcheck.go

```diff
@@ -66,7 +66,20 @@ func main() {
 		log.Fatal("-new is obligatory")
 	}
 
-	results, err := benchcheck.StatModule(*mod, *oldRev, *newRev)
+	runBench := benchcheck.DefaultRunBench
+	var customizedBenchCmd []string
+
+	for i, v := range os.Args {
+		if v == "--" {
+			customizedBenchCmd = os.Args[i+1:]
```

Comment:
```suggestion
			customizedBenchCmd = os.Args[i+1:]
			break
```

because if the user-supplied command also contains a `--` it's going to use the last one, which seems wrong. Example:
```
benchcheck -mod A -old B -new C -- wrapper -flag -- go test ...
```